src/shared/interfaces/User.ts

```diff
@@ -1,6 +1,10 @@
 export interface User {
-    username: string;
     email: string;
-    password: string;
+    role: string; 
+    firstName?: string; 
+    lastName?: string; 
+    city: string;
+    isDeleted: boolean;
+    likedEvents: string[];
```

Comment:
The same question as createdEvents - Here finally is it going to be an array of strings or array of objects as we've discussed? And isn't it better to be optional?